client/src/pages/SignIn/SignIn.js

```diff
@@ -0,0 +1,119 @@
+// import { Header } from "../components/Header/Header";
+// import React from "react"
+
+
+
+
+
+
+
+// export const SignIn = () => {
+
+//   const [name, setNAme] = useState("GFG");
+
+//   const handleChange = () => ();
+
+
+//   return(
+//     <div>
+//       <Header title="Sign In" />
+//       <form>
+//         <input value={name} onChange={handleChange}/>
+//       </form>
+//       <h1>hello</h1>
+//     </div>
+    
+//   )
+// };
+
+import { useState } from 'react';
+
+export function SignIn() {
+
+// States for registration
+const [email, setEmail] = useState('');
+const [password, setPassword] = useState('');
+
+// States for checking the errors
+const [submitted, setSubmitted] = useState(false);
+const [error, setError] = useState(false);
+
+
+// Handling the email change
+const handleEmail = (e) => {
+	setEmail(e.target.value);
+	setSubmitted(false);
+};
+
+// Handling the password change
+const handlePassword = (e) => {
+	setPassword(e.target.value);
+	setSubmitted(false);
+};
+
+// Handling the form submission
+const handleSubmit = (e) => {
+	e.preventDefault();
+	if (email === '' || password === '') {
+	setError(true);
+	} else {
+	setSubmitted(true);
+	setError(false);
+	}
+};
+
+// Showing success message
+const successMessage = () => {
```

Comment:
Can this be changed to show the homepage or something once the log in is successful?

Reply:
yes, we could add a link when we connect to the database